CHANGELOG.md

```diff
@@ -13,11 +13,13 @@
 * **lint** will now fail on `LOG` usage in python files.
 * Fixed an issue where **pre-commit** running when there was no change to the files
 * Fixed an issue where errors in **validate** were logged as `info`.
+* Fixed an issue where the **unify** command failed to unify a integration with a module, in cases where the cwd was not the root of the content repository.
```

Comment:
`with a module`, does it mean `which uses an API module`?

Reply:
Right.
`* Fixed an issue where the **unify** command failed to unify a integration with an API module, in cases where the cwd was not the root of the content repository.`
It's OK?

Reply:
```suggestion
* Fixed an issue where the **unify** command failed on integrations using an API a module, when not called from the content root.
```